pkg/subnet/local.go

```diff
@@ -224,7 +224,7 @@ func (d *LocalDeployer) doDeploy(chain string, chainGenesis []byte, genesisPath
 	firstURL := endpoints[0]
 	tokenName := d.app.GetTokenName(chain)
 
-	ux.Logger.PrintToUser("Metamask connection details (any node URL from above works):")
+	ux.Logger.PrintToUser("Browser Extension connection details (any node URL from above works):")
```

Comment:
Just "Browser Extension" sounds very general, should we mention Core? E.g. "Avalanche Core Browser Extension connection...."

Reply:
The RPC URL works with any browser extension including Core and Metamask for now. It could work with any other similar new browser extensions in the future.